aiida_cp2k/utils/workchains.py

```diff
@@ -27,9 +27,13 @@ def merge_dict(dct, merge_dct):
     :param merge_dct: dct merged into dct (overwrites dct data if in both)
     :return: None
     """
-    import collections
+    # Consider change of the collections interface in Python 3.10 and keep the code backwards compatible
+    try:
+        from collections.abc import Mapping  # pylint: disable=deprecated-class
+    except ImportError:
+        from collections import Mapping  # pylint: disable=deprecated-class
```

Comment:
`collections.abc` was introduced in Python 3.3. I think it is safe to just do this:
```suggestion
    from collections import Mapping
```

Reply:
@yakutovicha I guess you mean
```
from collections.abc import Mapping
```

Reply:
yep, sorry :)